### PR TITLE
Update Cygwin package list for SITL binaries

### PR DESCRIPTION
**Pull request by TunaLobster**
DELAY MERGE for ArduPilot/ardupilot#17083

The previous changes to this page went in before the PR was merged. That's on me for not saying that this was a delayed merge.

### CODE REVIEW

**Comment by TunaLobster** [comment]
I can build locally with WSL 1, 18.04, Sphinx 4.0.1, and tweaking common_conf.py. CI seems to be failing.

**Comment by Hwurzburg** [comment]
Ci fails everything....for the last few days...

**Comment by stephendade** [comment]
You'll also need to update the "Set up directories/paths and extra packages in Cygwin" section to have the updatled ``ln`` commands.

**Comment by TunaLobster** [comment]
Thanks Stephen! Rebased to get the CI fixes.

File: dev/source/docs/building-setup-windows-cygwin.rst [hunk 1]
```diff
@@ -17,74 +17,73 @@ Install Cygwin
 --------------
 
 #. Open a web browser to `www.cygwin.com/install.html <https://www.cygwin.com/install.html>`__ and download  "setup-x86.exe"
```
Comment (stephendade) [inline dev/source/docs/building-setup-windows-cygwin.rst:19]: Update to ``setup-x86_64.exe``

**Review by stephendade — CHANGES_REQUESTED** [review]

File: dev/source/docs/building-setup-windows-cygwin.rst (hunk 1 above)
Comment (TunaLobster) [reply dev/source/docs/building-setup-windows-cygwin.rst]: Fixed!

**Comment by stephendade** [comment]
Great - all good from me. LGTM

**Pull request merged**